src/arc_spice/eval/classification_error.py

```diff
@@ -8,6 +11,10 @@ def aggregate_score(probs: torch.Tensor) -> torch.Tensor:
     return 1 - torch.mean(distance)
 
 
+def zero_one_loss_ceil(y_target, y_pred):
```

Comment:
I think this is no longer really the zero_one_loss if we're doing a ceil, it's just a ==, so we can remove? or keep it as it was for use over a set of results (rather than single row)

Reply:
will remove